etip/trackers/models.py

```diff
@@ -81,24 +81,38 @@ def clean_fields(self, exclude=None):
                 {'code_signature': "Must be a valid regex."}
             )
 
+    def any_signature_collision(self):
+        trackers = Tracker.objects.all().exclude(id=self.id)
+        for t in trackers:
+            if self.__same_code_signature(t) \
+                    or self.__same_network_signature(t):
+                return True
+        return False
+
     def code_signature_collision(self):
         collisions = []
         trackers = Tracker.objects.all().exclude(id=self.id)
         for t in trackers:
-            if re.search(self.code_signature, t.code_signature) \
-                    and len(self.code_signature) > self.MIN_SIGNATURE_SIZE:
+            if self.__same_code_signature(t):
                 collisions.append(t.name)
         return collisions
 
     def network_signature_collision(self):
         collisions = []
         trackers = Tracker.objects.all().exclude(id=self.id)
         for t in trackers:
-            if re.search(self.network_signature, t.network_signature) \
-                    and len(self.network_signature) > self.MIN_SIGNATURE_SIZE:
+            if self.__same_network_signature(t):
                 collisions.append(t.name)
         return collisions
 
+    def __same_network_signature(self, tracker):
+        return re.search(self.network_signature, tracker.network_signature) \
```

Comment:
I'm wondering if this would be more readable the following way (or something similar):

```
         has_same_signature = re.search(self.network_signature, tracker.network_signature) \
            and len(self.network_signature) > self.MIN_SIGNATURE_SIZE
         return has_same_signature
```
WDYT?

Reply:
Don't you think that changing the method name could be enough? Like `_has_same_network_signature`. With that kind of renaming, I don't think setting a variable to the same thing can increase readability :thinking:

Reply:
Sure, that makes sense :).